### PR TITLE
store/tikv/gcworker: add manual gc back for some kv api users

### PR DESCRIPTION
**Pull request by wentaoxu**
some users which use kv api need to use this func to do gc.
PTAL @siddontang @zhangjinpeng1987 @tiancaiamao @coocood

### CODE REVIEW

**Comment by coocood** [comment]
@wentaoxu 
We can create a `GCWorker` inside `RunGCJob`.

**Comment by wentaoxu** [comment]
@coocood , we need  to execute sql query if we create `gcworker`, but for `kv` api users, they may not have the sql mudule.

**Comment by coocood** [comment]
@wentaoxu 
I mean manualy `new(GCWorker)` and assign the needed fields to it.
Then we the change would be minimum.

**Comment by wentaoxu** [comment]
@coocood ok

**Comment by wentaoxu** [comment]
PTAL @coocood

File: store/tikv/gcworker/gc_worker.go [hunk 1]
```diff
@@ -911,6 +915,28 @@ func (w *GCWorker) saveValueToSysTable(key, value string, s session.Session) err
...
+	gcWorker := &GCWorker{
+		store: s,
+		uuid:  identifier,
+	}
+
+	err := gcWorker.resolveLocks(ctx, safePoint)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	err = gcWorker.doGCManual(ctx, safePoint)
```
Comment (coocood) [inline store/tikv/gcworker/gc_worker.go]: How about calling `doGCInternal` directly?

Comment (wentaoxu) [reply store/tikv/gcworker/gc_worker.go]: ok

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM
/run-all-tests

**Pull request merged**